app/assets/stylesheets/scaffolds.scss

```diff
@@ -9,8 +9,8 @@ body {
 
 p, ol, ul, td {
   font-family: verdana, arial, helvetica, sans-serif;
-  font-size: 13px;
-  line-height: 18px;
+  font-size: 18px;
+  line-height: 20px;
```

Comment:
I'm not sure whether you intended to change these for the whole website or not. Right now it effects the events view and the header. If you did then you should be careful that it does not have unintended effects in certain views that use those values, which is quite a lot. Otherwise it may be safer to make a class for whatever portion you want to change and put the font size and height changes there.

Reply:
yeah this should be changed^ makes the front page look very sloppy.